Rakefile

```diff
@@ -128,14 +128,18 @@ task :c2m_check_version_on_dir, [:last_checked_b4_date, :storage_root, :profile]
   root = args[:storage_root].to_sym
   storage_dir = "#{Settings.moab.storage_roots[root]}/#{Settings.moab.storage_trunk}"
   last_checked = args[:last_checked_b4_date].to_sym
-
-  if args[:profile] == 'profile'
-    puts "When done, check log/profile_c2m_check_version_on_dir[TIMESTAMP].txt for profiling details"
-    CatalogToMoab.check_version_on_dir_profiled(last_checked, storage_dir)
-  elsif args[:profile].nil?
-    CatalogToMoab.check_version_on_dir(last_checked, storage_dir)
+  begin
+    if args[:profile] == 'profile'
+      puts "When done, check log/profile_c2m_check_version_on_dir[TIMESTAMP].txt for profiling details"
+      CatalogToMoab.check_version_on_dir_profiled(last_checked, storage_dir)
+    elsif args[:profile].nil?
+      CatalogToMoab.check_version_on_dir(last_checked, storage_dir)
+    end
+    puts "#{Time.now.utc.iso8601} Catalog to Moab version check on #{storage_dir} is done."
+  rescue TypeError, ArgumentError
+    p "You've entered an incorrect timestamp format #{last_checked}."
+    p "Please enter correct timestamp format (2018-02-01T18:54:48)"
```

Comment:
is this UTC time?  We probably want to say so here ... and imply so with the Z showing (Z on the end implies UTC).